.mergify.yml

```diff
@@ -7,6 +7,22 @@ pull_request_rules:
       dismiss_reviews:
         approved: true
         changes_requested: false
+  - name: ask to resolve conflict
+    conditions:
+      - conflict
+    actions:
+      comment:
+        message: This pull request is now in conflicts. Could you fix it? 🙏
```

Comment:
Maybe: This pull request now has conflicts with the target branch. Could you please rebase and force push the corrected changes? :pray: